main.py

```diff
@@ -192,8 +209,25 @@ def is_valid_tag(tag: Optional[str]) -> bool:
 
 def get_cover_art(filename: str) -> Optional[File]:
     # Get image data as bytes
-    tags = TinyTag.get(filename, image=True)
-    image_data = tags.get_image()
+    try:
+        image_data = None
+        mt = MutagenFile(filename)
+        if isinstance(mt, ID3FileType):
+            if "APIC:cover" in mt.tags:
+                image_data = mt.tags["APIC:cover"].data
```

Comment:
I don't think the part after APIC: is necessary or even sufficient for it's associated tag to be cover art. From what I understand, "APIC:(description)" just denotes any Attached PICture frame with <description> being a unique identifier for the image, but doesn't determine its purpose. It's the "type" field within the APIC frame that marks an image as a cover ([see Wikipedia](https://en.wikipedia.org/wiki/ID3#ID3v2_embedded_image_extension)). The program I use to tag mp3s and give them covers, Kid3, seems to put the original filename as the APIC description and so this code does not pick it up.

So I think the proper way to detect a cover art is:
1) Search through all tags `tag` in `mt.tags` starting with "APIC:"
2) If the [APIC object](https://mutagen.readthedocs.io/en/latest/api/id3_frames.html?highlight=apic#mutagen.id3.APIC) `mt.tags[tag]`'s type field is [PictureType.COVER_FRONT](https://mutagen.readthedocs.io/en/latest/api/id3.html?highlight=picturetype#mutagen.id3.PictureType), then we have found our cover art and break the loop.